pkgs/top-level/python-packages.nix

```diff
@@ -13872,6 +13872,7 @@ self: super: with self; {
       if pkgs.config.cudaSupport
       then pkgs.magma-cuda-static
       else pkgs.magma;
+    cudaPackages = pkgs.cudaPackages_12;
```

Comment:
does it not work with `pkgs.cudaPackages`? at some point we should really just upgrade `cudaPackages`, but in the meantime this will create a duplication of cuda installations on folks' machines

Reply:
No I thought it was needing it but it works fine with cuda 11 as well. I reverted that change.